packages/test-runner-selenium/test/seleniumLauncher.test.ts

```diff
@@ -11,8 +11,8 @@ async function startSeleniumServer() {
   try {
     await selenium.install({
       drivers: {
-        chrome: { version: '96.0.4664.18' },
-        firefox: { version: '0.30.0' },
+        chrome: { version: 'latest' },
+        firefox: { version: 'latest' },
```

Comment:
Would this be crazy instead: 
```
  try {
    await selenium.install({
      drivers: {
        chrome: { version: 'latest' },
        firefox: { version: 'latest' },
      },
    });
  } catch (err) {
    try {
      await selenium.install({
        drivers: {
          chrome: { version: '97.0.4692.99' },
          firefox: { version: 'latest' },
        },
      });
    } catch (err) {
      console.error('Error occurred when installing selenium.');
      throw err;
    }
  }
```
Maybe we could parse the `err` for a more nuanced second try?

Reply:
I'd like to avoid hardcoding. Build passed now, let's reconsider this if the problem happens again.